cognite/client/_api/geospatial.py

```diff
@@ -491,6 +491,88 @@ def update_features(
             ),
         )
 
+    def list_features(
+        self,
+        feature_type_external_id: str,
+        filter: Optional[Dict[str, Any]] = None,
+        properties: Dict[str, Any] = None,
+        limit: int = 100,
+        allow_crs_transformation: bool = False,
+    ) -> FeatureList:
+        """`List features`
+        <https://docs.cognite.com/api/v1/#operation/listFeatures>
+
+        This method allows to filter all features.
+
+        Args:
+            feature_type_external_id: the feature type to list features for
+            filter (Dict[str, Any]): the list filter
+            limit (int, optional): Maximum number of features to return. Defaults to 25. Set to -1, float("inf") or None
+                to return all features.
+            properties (Dict[str, Any]): the output property selection
+            allow_crs_transformation: If true, then input geometries will be transformed into the Coordinate Reference
```

Comment:
Might be a copy paste error here. There are no input geometries with the /list endpoint